airbyte-integrations/connectors/source-alpha-vantage/source_alpha_vantage/spec.yaml

```diff
@@ -0,0 +1,53 @@
+documentationUrl: https://docsurl.com
```

Comment:
Correct to the connector doc url, see other connectors.

Reply:
Done